serverjs/cubefn.js

```diff
@@ -243,7 +243,7 @@ function CSVtoCards(csvString, carddb) {
     'collector number': collectorNumber,
     status,
     finish,
-    maybeboard,
+    maybeboard = 'false',
```

Comment:
I believe this should be `maybeboard: 'false'`

Reply:
This form `const { variableNames } of data` is the ["for of" variation of Destructuring Assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#for_of_iteration_and_destructuring).

Using a `:` colon here would be the syntax for [assigning new variable names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assigning_to_new_variable_names), ie renaming a property.

`const { foo: bar } = { foo: 'baz' } // -> foo === undefined, bar === 'baz'`

Using an assignment operator `=` provides a [default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_values_2).

`const { foo = 'baz' } = { foo: 'bar' }; // foo === 'bar'`
`const { foo = 'baz' } = { bar: 4 }; // foo === 'baz'`

On the very next line, there's a colon being used to assign a new variable name.

https://github.com/dekkerglen/CubeCobra/blob/8840550168801628ed5204cdd2612edda389370b/serverjs/cubefn.js#L237-L253

Reply:
I don't really have a problem with this solution, but maybe it would be more straightforward to just chuck `if (maybeboard && ...` into the check instead to avoid the possible confusion of mixing these two together?

Reply:
Or, as I just realized, leave your proposed check as is. If we're doing the type checking, we don't actually need a default, and vice versa.